tests/integrationv2/test_renegotiate.py

```diff
@@ -193,7 +193,8 @@ def basic_reneg_test(managed_process, cipher, curve, certificate, protocol, prov
 @pytest.mark.parametrize("certificate", MINIMAL_TEST_CERTS, ids=get_parameter_name)
 @pytest.mark.parametrize("protocol", TEST_PROTCOLS, ids=get_parameter_name)
 @pytest.mark.parametrize("provider", [OpenSSL], ids=get_parameter_name)
-def test_s2n_client_ignores_openssl_hello_request(managed_process, cipher, curve, certificate, protocol, provider):
+@pytest.mark.parametrize("other_provider", [S2N], ids=get_parameter_name)
```

Comment:
If this does have to be reverted, should we also add S2N as a `other_provider` to the other tests in this file?

Reply:
reverted- and no, other_provider not needed since we sorted out the weird xdist failure.